scheduler_failover_controller/failover/failover_controller.py

```diff
@@ -160,25 +172,38 @@ def is_scheduler_running(self, host):
             # todo: If there's more then one scheduler running this should kill off the other schedulers. MIGHT ALREADY BE HANDLED. DOUBLE CHECK.
 
             is_running = active_list_length > 0
+            if (is_running):
+                url = "http://" + \
+                    str(host) + ":" + str(conf.get("webserver",
```

Comment:
There's no gaurantee that the `host` value will have a webserver running on it. Lets use the webserver url value from the airflow.cfg file and ensure that's a requirement of the controller in README.md